### PR TITLE
codeowner: add code owner

### PR DESCRIPTION
**Pull request by tyxia**
Per discussion https://github.com/envoyproxy/envoy/pull/36619#issuecomment-2417696028

### CODE REVIEW

**Review by alyssawilk — COMMENTED** [review]
LGTM for senior maintainer stamp but will defer to @kyessenov

**Review by kyessenov — APPROVED** [review]

**Pull request merged**
